### PR TITLE
Fixes for issue 111 and 112

### PR DESCRIPTION
**Pull request by lsces**
These add files only to the demo package and should provide a working demo for the composer installed version as well as the option to run without composer installed.

### CODE REVIEW

File: demo/AutoLoader.php [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<?php
```
Comment (aik099) [inline demo/AutoLoader.php]: That's the original auto-loader that was used before Composer integration?

**Comment by aik099** [comment]
I've created separate issues for autoloader and for missing annotation for demo package, because 2 different problems are solved.

Please submit as 2 different PRs.

File: demo/standalone.php [hunk 2]
```diff
@@ -0,0 +1,371 @@
+<?php
```
Comment (aik099) [inline demo/standalone.php]: I guess this is a copy/paste of demo script that uses different autoloader.

Please instead add commented-out fragment using different autoloader in original demo file.

Also please run `wiki-doc.php` file to regenerate documentation page using that file.

**Comment by aik099** [comment]
This PR would stay for autoloader changes and new PR would be for annotation adding.

File: demo/AutoLoader.php (hunk 1 above)
Comment (lsces) [inline demo/AutoLoader.php]: As far as I could see the original demo had an in-line loader and the switch to PSR4 style loading only happened after composer had replaced the various options used prior to that? One of the reasons I had a problem initially.

Comment (aik099) [inline demo/AutoLoader.php]: So the answer is no. Isn't one simple function enough to implement PSR-4 autoloader? The autoloader you've added in this PR is huge.

**Comment by lsces** [comment]
This has been replaced by individual PR's

**Pull request closed without merging**